<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="helloWorld_Button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hello world!</string>
    </property>
   </widget>
   <widget class="QListWidget" name="character_list">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>161</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>60</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="health_label">
       <property name="text">
        <string>Health</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="energy_label">
       <property name="text">
        <string>Energy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="level_label">
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="defence_label">
       <property name="text">
        <string>Defence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dexterity_label">
       <property name="text">
        <string>Dexterity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="constitution_label">
       <property name="text">
        <string>Constitution</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>110</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="name_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="health_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="energy_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="level_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="defence_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="dexterity_lineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="constitution_lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
